### PR TITLE
Feature crud cluster activity partner project #106

### PR DESCRIPTION
**Pull request by lkolacz**
##### Feature list
* CRUD cluster acitvity

* CRUD partner projects

##### Progress checker
- [x] Django

- [x] Django test cases

### CODE REVIEW

File: django_api/django_api/apps/cluster/views.py [hunk 1]
```diff
@@ -97,3 +102,87 @@ def post(self, request, *args, **kwargs):
...
+        Update on ClusterActivity model
+        :return: ClusterActivity serializer data
+        """
+        if 'id' in self.request.data.keys():
+            serializer = ClusterActivitySerializer(
+                instance=self.get_instance(self.request, pk=pk),
+                data=self.request.data
+            )
+        else:
+            return Response({"id": "This filed is reqired!"}, status=statuses.HTTP_400_BAD_REQUEST)
```
Comment (n1207n) [inline django_api/django_api/apps/cluster/views.py]: Typo here again :(

File: django_api/django_api/apps/cluster/tests/test_views.py [hunk 2]
```diff
@@ -113,3 +113,117 @@ def test_read_cluster_objective(self):
         response = self.client.get(url, format='json')
         self.assertTrue(status.is_success(response.status_code))
         self.assertEquals(response.data['count'], 1)
+
+
+class TestClusterActivityAPIView(BaseAPITestCase):
+
+    @property
+    def data(self):
```
Comment (lkolacz) [inline django_api/django_api/apps/cluster/tests/test_views.py]: done

File: django_api/django_api/apps/cluster/views.py (hunk 1 above)
Comment (lkolacz) [inline django_api/django_api/apps/cluster/views.py]: done

File: django_api/django_api/apps/cluster/tests/test_views.py [hunk 3]
```diff
@@ -113,3 +113,117 @@ def test_read_cluster_objective(self):
...
+        response = self.client.post(url, data=self.data, format='json')
+
+        self.assertTrue(status.is_success(response.status_code))
+        self.assertEquals(response.data['id'], (last.id+1))
+        created_obj = ClusterActivity.objects.get(id=response.data['id'])
+        self.assertEquals(created_obj.title, self.data["title"])
+        self.assertEquals(created_obj.frequency, FREQUENCY_LEVEL.weekly)
+        self.assertEquals(ClusterActivity.objects.all().count(),  base_count + 1)
+
+    def test_get_cluster_activity(self):
```
Comment (lkolacz) [inline django_api/django_api/apps/cluster/tests/test_views.py]: done

File: django_api/django_api/apps/cluster/serializers.py [hunk 4]
```diff
@@ -1,6 +1,22 @@
 from rest_framework import serializers
 from core.common import FREQUENCY_LEVEL
-from .models import ClusterObjective
+from .models import ClusterObjective, ClusterActivity, Cluster
+
+
+class ClusterSimpleSerializer(serializers.ModelSerializer):
+
+    id = serializers.SerializerMethodField()
```
Comment (lkolacz) [inline django_api/django_api/apps/cluster/serializers.py]: I understand Your point of view... I mean really! But this is something that will be forced by Rafał... so If You don't agree then Rafał will push on Sumit to force pk as str...
Soo.. I can help Rafał to make them front end non bug development or not... I decided to help!
If You disagree then this shouldn't be a blocker to merge ... especially that there is a lot of place like this.. and should be done in separated issue..
Thx!

Comment (lkolacz) [inline django_api/django_api/apps/cluster/serializers.py]: Done ;-)

**Pull request merged**